### PR TITLE
Part2

### PR DESCRIPTION
**Pull request by xyzdelete**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -127,297 +169,519 @@ struct CarWash
...
+{
+    if (subjectName == "Programming")
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
```
[the remark this answers is not in the document]
Comment (xyzdelete) [reply main.cpp:196]: Fixed.

File: main.cpp [hunk 2]
```diff
@@ -127,297 +169,519 @@ struct CarWash
...
+
+bool Monitor::goIntoSleepMode()
+{
+    return true;
+}
+
+int displayWarning()
+{
+    return 0;
+}
```
[the remark this answers is not in the document]
Comment (xyzdelete) [reply main.cpp:249]: Fixed. Idk how I missed this.

File: main.cpp [hunk 3]
```diff
@@ -127,297 +169,519 @@ struct CarWash
...
+            if (numOfCombustionChambersToCheck > 0)
+            {
+                    if (numOfExhaustNozzlesToCheck > 0)
+                    {
+                        return true;
+                    }
+            }
+    }
+    return false;
+}
```
[the remark this answers is not in the document]
Comment (xyzdelete) [reply main.cpp:322]: Fixed. Yeah, I didn't spot it.

File: main.cpp [hunk 4]
```diff
@@ -127,297 +169,519 @@ struct CarWash
...
+{
+    if (fuel == "Liquid fuel")
+    {
+        return amountOfFuel;
+    }
+    else
+    {
+        return 0;
+    }
+}
```
[the remark this answers is not in the document]
Comment (xyzdelete) [reply main.cpp:339]: Fixed.

File: main.cpp [hunk 5]
```diff
@@ -127,297 +169,519 @@ struct CarWash
...
+    {
+        someEngine.produceThrust();
+        someEngine.burnFuel();
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
```
[the remark this answers is not in the document]
Comment (xyzdelete) [reply main.cpp:355]: Fixed.

File: main.cpp [hunk 6]
```diff
@@ -127,297 +169,519 @@ struct CarWash
...
+{
+    if (mHz != 99999)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
```
[the remark this answers is not in the document]
Comment (xyzdelete) [reply main.cpp:490]: Fixed and fixed.

File: main.cpp [hunk 7]
```diff
@@ -127,297 +169,519 @@ struct CarWash
...
+{
+    if (instructionType != 0)
+    {
+        return 1;
+    }
+    else
+    {
+        return 0;
+    }
+}
```
[the remark this answers is not in the document]
Comment (xyzdelete) [reply main.cpp:540]: Fixed.

File: main.cpp [hunk 8]
```diff
@@ -127,297 +169,519 @@ struct CarWash
...
-    
+
+void SolidStateDrive::store(std::string someData)
+{
+    if (someData != "")
+    {
+        return;
+    }
+    return;
+}
```
[the remark this answers is not in the document]
Comment (xyzdelete) [reply main.cpp:563]: Fixed.

File: main.cpp [hunk 9]
```diff
@@ -127,297 +169,519 @@ struct CarWash
...
+{
+    if (encryptionType != 0)
+    {
+        return "Encrypted: " + dataToDecrypt;
+    }
+    else
+    {
+        return dataToDecrypt;
+    }
+}
```
[the remark this answers is not in the document]
Comment (xyzdelete) [reply main.cpp:587]: Fixed.

File: main.cpp [hunk 10]
```diff
@@ -127,297 +169,519 @@ struct CarWash
...
+{
+    if (standard != "")
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
```
[the remark this answers is not in the document]
Comment (xyzdelete) [reply main.cpp:627]: Fixed.

File: main.cpp [hunk 11]
```diff
@@ -127,297 +169,519 @@ struct CarWash
...
+{
+    if (programName != "")
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
```
[the remark this answers is not in the document]
Comment (xyzdelete) [reply main.cpp:668]: Fixed.

File: main.cpp [hunk 12]
```diff
@@ -127,297 +169,519 @@ struct CarWash
...
+        module.connectToTheInternet();
+        module.sendData(data);
+        
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
```
[the remark this answers is not in the document]
Comment (xyzdelete) [reply main.cpp:684]: Fixed.

**Review by matkatmusic — CHANGES_REQUESTED** [review]

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**